generic_routines.sh

```diff
@@ -186,7 +186,7 @@ applyFixes() {
     fixesFilePath=$2
     fileTypeToBeFixed=$3
 
-    echo "NOTE: Fix will not be applied in lines of $fixesFile that miss a tab character"
+    echo "NOTE: Fix will not be applied in lines of fixesFile that miss a tab character"
```

Comment:
Won't this just print fixesFile instead of value of `$fixesFile`?

Reply:
The var $fixesFile is never used. Probably we carried it over from previous PRs, without noticing so fast as it was not producing error